merge.py

```diff
@@ -1,98 +1,196 @@
-"""Mergebot is a script which talks to GitHub and submits all ready pull requests.
-
-Mergebot talks to a specified GitHub project and watches for @mentions for its account.
-Acceptable commands are:
-  @<mergebot-name> merge
+"""merge contains classes which will merge work into SCM systems.
 """
+import Queue
 from subprocess import call
-import glob
-import requests
 import sys
-import time
+import github_helper
+
+APACHE_URL = 'https://git-wip-us.apache.org/repos/asf/{}.git'
```

Comment:
It would be nice to have some comments explaining what will go in each of the holes, and/or using the named substitution syntax. So:

```
APACHE_URL = 'https://git-wip-us.apache.org/repos/asf/{repository}.git'
...
APACHE_URL.format(repository=...)
```

It will make both these format strings, and their uses, easier to read.